x/ibc/04-channel/keeper/packet_test.go

```diff
@@ -644,9 +661,18 @@ func (suite *KeeperTestSuite) TestAcknowledgementExecuted() {
 			err := suite.chainA.App.IBCKeeper.ChannelKeeper.AcknowledgementExecuted(suite.chainA.GetContext(), chanCap, packet)
 			pc := suite.chainA.App.IBCKeeper.ChannelKeeper.GetPacketCommitment(suite.chainA.GetContext(), packet.GetSourcePort(), packet.GetSourceChannel(), packet.GetSequence())
 
+			channelA, _ := suite.chainA.App.IBCKeeper.ChannelKeeper.GetChannel(suite.chainA.GetContext(), packet.GetSourcePort(), packet.GetSourceChannel())
+			sequenceAck, _ := suite.chainA.App.IBCKeeper.ChannelKeeper.GetNextSequenceAck(suite.chainA.GetContext(), packet.GetSourcePort(), packet.GetSourceChannel())
+
 			if tc.expPass {
 				suite.NoError(err)
 				suite.Nil(pc)
+
+				if channelA.Ordering == types.ORDERED {
+					suite.Require().Equal(packet.GetSequence()+1, sequenceAck, "sequence not incremented in ordered channel")
+				} else {
+					suite.Require().Equal(packet.GetSequence(), sequenceAck, "sequence incremented for UNORDERED channel")
```

Comment:
```suggestion
					suite.Require().Equal(uint64(1), sequenceAck, "sequence incremented for UNORDERED channel")
```
nit: in case a future test case uses a packet of sequence > 1

Reply:
Wait isn't that why we would want it to be the original way in the first place??

Reply:
I'm not sure I follow your response, but in UNORDERED channels next sequence ack is never incremented